test/core/ajax.js

```diff
@@ -559,6 +559,34 @@ describe('Core htmx AJAX Tests', function() {
     values.should.deep.equal({ c1: ['cb1', 'cb3'], c2: 'cb5', c3: 'cb6' })
   })
 
+  it('properly handles radio inputs', function() {
```

Comment:
This is essentially just a cut-down clone of the function above which tests checkboxes in detail, to cover the `shouldInclude` logic which applies to both input types:

`if (elt.type === 'checkbox' || elt.type === 'radio') {
      return elt.checked
    }`